src/Microsoft.DotNet.Helix/Sdk/tools/xharness-runner/xharness-event-processor.py

```diff
@@ -126,7 +126,7 @@ def analyze_operation(command: str, platform: str, device: str, is_device: bool,
     global retry, reboot, android_connectivity_verified
 
     # Kill the simulator when we fail to launch the app
-    if exit_code == 80: # APP_CRASH
+    if exit_code == 80 and not is_device: # APP_CRASH
```

Comment:
I actually think Android phones benefit from this so let's do it for all but Apple devices?

```suggestion
    # Apps crashing can be infra failures apart form Apple devices where retries can be costly
    if exit_code == 80 and platform != "apple" and is_device: # APP_CRASH
```